app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -162,4 +201,53 @@ private String createSignedDocument(ConsentSignature consent, StudyConsent study
         html = html.replace("@@signing.date@@", signingDate);
         return html;
     }
+    
+    private String createInlineParticipantRoster(List<StudyParticipant> participants) {
+        StringBuilder sb = new StringBuilder("There are "+participants.size()+" users enrolled in this study:"+NEWLINE);
+        for (int i=0; i < participants.size(); i++) {
+            StudyParticipant participant = participants.get(i);
+            
+            sb.append(NEWLINE).append(participant.getEmail()).append(" (");
+            if (participant.getFirstName() == null && participant.getLastName() == null) {
+                sb.append("No name given");
+            } else if (participant.getFirstName() != null && participant.getLastName() != null) {
+                sb.append(participant.getFirstName()).append(" ").append(participant.getLastName());
+            } else if (participant.getFirstName() != null) {
+                sb.append(participant.getFirstName());
+            } else if (participant.getLastName() != null) {
+                sb.append(participant.getLastName());
+            }
+            sb.append(")"+NEWLINE);
+            if (participant.getPhone() != null) {
+                sb.append("Phone: ").append(participant.getPhone()).append(NEWLINE);    
+            }
+        }
+        return sb.toString();
+    }
+    
+    private String createParticipantCSV(List<StudyParticipant> participants) {
+        StringBuilder sb = new StringBuilder();
+        append(sb, "Email");
+        append(sb, "First Name");
+        append(sb, "Last Name");
+        append(sb, "Phone");
+        sb.append(NEWLINE);
+        for (int i=0; i < participants.size(); i++) {
+            StudyParticipant participant = participants.get(i);
+            append(sb, participant.getEmail());
+            append(sb, participant.getFirstName());
+            append(sb, participant.getLastName());
+            append(sb, participant.getPhone());
+            sb.append(NEWLINE);
+        }
+        return sb.toString();
+    }
+    
+    private void append(StringBuilder sb, String value) {
+        if (value != null) {
+            sb.append(escapeCsv(value)).append(COMMA_DELIMITER);
+        } else {
+            sb.append("").append(COMMA_DELIMITER);
+        }
+    }
```

Comment:
I tested this and the escaping library didn't seem to do enough to protect the cdv format. Switching to tabs.